Example/Tests/Security/CryptographyTests.swift

```diff
@@ -101,7 +101,7 @@ class CryptographyTests: XCTestCase {
         var map = [AnyHashable: Any]()
 
         for _ in 0..<10000 where map.count < 256 {
-            let rand = DMECrypto.getRandomUnsignedCharacters(1)
+            let rand = DMECryptoUtilities.getRandomBytes(withLength: 1)
```

Comment:
Don;t forget to update the tests with new signature for the `randomBytes(withLength:)` function